mod-consortia/src/main/resources/thunderjet/mod-consortia/features/tenant.feature

```diff
@@ -374,14 +375,23 @@ Feature: Tenant object in mod-consortia api tests
     And request { id: '#(centralTenant)', code: 'ABD', name: 'Central tenants name', isCentral: true}
     When method PUT
     Then status 200
-    And match response == { id: '#(centralTenant)', code: 'ABD', name: 'Central tenants name', isCentral: true }
+    And match response == { id: '#(centralTenant)', code: 'ABD', name: 'Central tenants name', isCentral: true, isDeleted: false }
 
     # update the tenants' name
     Given path 'consortia', consortiumId, 'tenants', centralTenant
     And request { id: '#(centralTenant)', code: 'ABD', name: 'Central tenants name updated', isCentral: true}
     When method PUT
     Then status 200
-    And match response == { id: '#(centralTenant)', code: 'ABD', name: 'Central tenants name updated', isCentral: true }
+    And match response == { id: '#(centralTenant)', code: 'ABD', name: 'Central tenants name updated', isCentral: true, isDeleted: false }
+
+  @Negative
+  Scenario: Attempt to change the centrality of tenant with isCentral = false
+    # try to update centrality of tenant
```

Comment:
this issue is recently found and fixed in scope of MODCON-130 pr in mod-consortia